docs/recipes.md

```diff
@@ -333,4 +333,33 @@ class DebugExampleApplication : ExampleApplication() {
         })
   }
 }
-```
\ No newline at end of file
+```
+
+## Using LeakCanary with obfuscated apps
+
+If obfuscation is turned on then leak traces will be obfuscated. It's possible to automatically deobfuscate leak traces by using a deobfuscation gradle plugin provided by LeakCanary.
+
+You have to add a plugin dependency in your root `build.gradle` file:
+
+```groovy
+dependencies {
+  classpath 'com.squareup.leakcanary:leakcanary-deobfuscation-gradle-plugin:${version}'
+}
```

Comment:
nit: those dependencies have to go into the `buildScript` section, right? If yes, then this mightt be more obvious:

```
buildscript {
  dependencies {
    classpath 'com.squareup.leakcanary:leakcanary-deobfuscation-gradle-plugin:${version}'
  }
}
```